validator/rpc/handlers_keymanager.go

```diff
@@ -839,3 +839,87 @@ func (s *Server) DeleteGasLimit(w http.ResponseWriter, r *http.Request) {
 	// we respond "not found".
 	httputil.HandleError(w, fmt.Sprintf("No gas limit found for pubkey %q", rawPubkey), http.StatusNotFound)
 }
+
+func (s *Server) GetGraffiti(w http.ResponseWriter, r *http.Request) {
+	ctx, span := trace.StartSpan(r.Context(), "validator.keymanagerAPI.GetGraffiti")
+	defer span.End()
+
+	if s.validatorService == nil {
+		httputil.HandleError(w, "Validator service not ready.", http.StatusServiceUnavailable)
+		return
+	}
+	rawPubkey, pubkey, ok := shared.HexFromRoute(w, r, "pubkey", fieldparams.BLSPubkeyLength)
+	if !ok {
+		return
+	}
+
+	graffiti, err := s.validatorService.GetGraffiti(ctx, bytesutil.ToBytes48(pubkey))
+	if err != nil {
+		httputil.HandleError(w, errors.Wrap(err, "graffiti was not set").Error(), http.StatusNotFound)
```

Comment:
Here the returned error is `graffiti was not set` even if the error is totally different right?

Reply:
Example, if I query the graffiti for a key non existing in our VC:
```
{
  "message": "graffiti was not set: rpc error: code = NotFound desc = Could not find validator index for public key 0xa17524bcf06d4fbf042e17dae68637368607f4fc9c29c104f49b12d1e0bed93fa3714f35ff05f4e518f789319c9b8f86",
  "code": 404
}
```

Having both `graffiti was not set` and `Could not find validator index for public key` is quite confusing IMO.